packages/common/src/types/index.ts

```diff
@@ -145,6 +145,41 @@ export type RoomInfo = {
   };
 };
 
+export type Pick = {
```

Comment:
@christianalfoni Maybe we should rename this type, since you also have the build in `Pick` generic
https://www.typescriptlang.org/docs/handbook/advanced-types.html#index-types

```ts
type Pick<T, K extends keyof T> = {
    [P in K]: T[P];
}
```